app/helpers/order_cycles_helper.rb

```diff
@@ -91,8 +91,6 @@ def validated_enterprise_options(enterprises, options={})
         elsif e.payment_methods.available.empty?
           disabled_message = I18n.t(:no_payment)
         end
-      elsif options[:confirmed] && !e.confirmed?
-        disabled_message = I18n.t(:unconfirmed)
```

Comment:
Just wondering if we need to replace the `confirmed` check here with an `activated` scope or something? These permissions behave a lot like scopes, in they are used to load lists of enterprises for select2 dropdowns etc. I think we want to make sure that enterprises that are owned by users who have not confirmed their email addresses are ignored by theses scopes? This is of course only for legacy data, as all new enterprises will have a confirmed owner by definition. What do you think?

Reply:
Bump

Reply:
@oeoeaio I feel a bit reluctant to replace a scope that represents "has never been verified/activated" with a scope that represents "has been verified and activated before but is changing owner". 

An enterprise can't be created or accessed by a non-verified user. The enterprise contact can't be changed to a non-verified user in the new UI. Do we want to kick an enterprise out of all scopes if they change owner and the owner has signed up with OFN but hasn't verified their email yet? What if the enterprise is in the middle of an order cycle?

Reply:
@Matt-Yorkley Do you have any objections to preventing non-verified users from owning Enterprises, in the same way we prevent non-verified users from being the contact? That feels safe to me, and precludes the possibility of the weird scope jumping you mentioned.

I must admit that I'm a bit confused about where we ended up in terms of auto-verifying pre-existing users (regardless of whether they have actually confirmed their email addresses). If we have decided to do that, perhaps that would negates the need for these scopes, since there will be no such thing as a legacy enterprise, all users with access to Enterprises will be verified by default.

Just trying to eliminate the possibility of ending up with broken data that will difficult to fix retrospectively...